api/PayrollProcessor.Web.Api/Features/Departments/DepartmentEmployeesGet.cs

```diff
@@ -31,9 +32,9 @@ public DepartmentEmployeesGet(IQueryDispatcher dispatcher)
             OperationId = "DepartmentEmployees.GetAll",
             Tags = new[] { "Employees", "Departments" })
         ]
-        public override Task<ActionResult<DepartmentEmployeesResponse>> HandleAsync([FromQuery] DepartmentEmployeesGetRequest request) =>
+        public override Task<ActionResult<DepartmentEmployeesResponse>> HandleAsync([FromQuery] DepartmentEmployeesGetRequest request, CancellationToken token) =>
```

Comment:
This is a required addition since the V2 of Endpoints.

Reply:
Nice addition of the cancellation token. I wonder who proposed that enhancement 🤔 😜

Reply:
Someone who was ok with me going through and updating all the methods 😁